Update IOS Facts documentation to say mebibytes (MiB) instead of megabits (Mb)

Troll3Killer opened the pull request:
##### SUMMARY
Updated the documentation for the ansible_net_memtotal_mb and ansible_net_memfree_mb return values to have the acronym for mebibytes (MiB) instead of megabits (Mb). In the source code it seems to be converting the values reported by the Cisco device from bits to mebibytes.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
ios_facts

##### ADDITIONAL INFORMATION
The source code for finding the unit conversion can be found here:
https://github.com/ansible-collections/cisco.ios/blob/main/plugins/module_utils/network/ios/facts/legacy/base.py#L151

```

KB-perByte commented:
@Troll3Killer thank you for your interest in contribution.
Just so you know, the .rst's are autogenerated from module documentation. The lines that are expected to be updated are -
https://github.com/ansible-collections/cisco.ios/blob/main/plugins/modules/ios_facts.py#L180
https://github.com/ansible-collections/cisco.ios/blob/main/plugins/modules/ios_facts.py#L184

Please consider updating the changes in the correct place and the CI would fix the .rst accordingly.
Also, add a changelog file to your PR.

Regards.

Troll3Killer commented:
@KB-perByte thank you for the feedback. 
I have updated the lines you linked to and added a changelog file as requested. Please let me know if there is anything else that is needed for this PR.
Sincerely,
Tyler
